com.rclone.ui.app.yml

```diff
@@ -0,0 +1,61 @@
+id: com.rclone.ui.app
+
+runtime: org.gnome.Platform
+runtime-version: '46'
+sdk: org.gnome.Sdk
+
+command: app
+finish-args:
+  - --socket=wayland # Permission needed to show the window
+  - --socket=fallback-x11 # Permission needed to show the window
+  - --device=dri # OpenGL, not necessary for all projects
+  - --share=ipc
+  - --share=network  # Network access
+  - --talk-name=org.freedesktop.Notifications  # Desktop Notifications
```

Comment:
what does it use for notifications and why doesn't it use the portals?

Reply:
Their documentation is wrong on many aspects. It's not Flathub's either.